spec/svg_spec.rb

```diff
@@ -1,7 +1,7 @@
 require 'rails_helper'
 
 RSpec.describe 'SVG files' do
-  Dir[Rails.root.join('**', '*.svg')].each do |svg_path|
+  Dir[Rails.root.join('**', '*.svg')].reject { |f| f['node_modules'] }.each do |svg_path|
```

Comment:
Yeah, there are some offenders in the `identity-style-guide` npm package so this is a temporary measure to exclude those for now. None of the images are being used within the app at this point. I will update to the syntax you provided... I wasn't sure the best way to handle excluding just that one folder.